project.clj

```diff
@@ -44,7 +44,7 @@
                  [puppetlabs/trapperkeeper-scheduler]
                  [puppetlabs/trapperkeeper-status]
                  [puppetlabs/kitchensink]
-                 [puppetlabs/ssl-utils]
+                 [puppetlabs/ssl-utils "0.8.2-SNAPSHOT"]
```

Comment:
Updating the clj-parent dep sounds good to me.  jvm-ssl-utils doesn't get revved that often.